### PR TITLE
Created TC1

### PR DESCRIPTION
**Pull request by Mars13O**

### CODE REVIEW

File: src/utils/TC1.java [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package utils;
+
+public class TC1 {
+    // new method
+    void method1(){
+
+    }
```
Comment (alfer216) [inline src/utils/TC1.java:7]: looks good, but can you make the method static?

Comment (Mars13O) [reply src/utils/TC1.java:7]: You right will do it

Comment (Mars13O) [reply src/utils/TC1.java:7]: updated

Comment (Natalija93) [reply src/utils/TC1.java:7]: just checking if i can review everything or only what has been requested to me

**Review by Natalija93 — APPROVED** [review]

**Pull request merged**